models/order.js

```diff
@@ -1,6 +1,7 @@
 import mongoose from 'mongoose';
 import { Schema } from 'mongoose';
 import constants  from '../src/constants';
+import Item from './item';
 
 const Entry = new Schema({
   item_id: { type: Schema.ObjectId, ref: 'Item' },
```

Comment:
Right, using references allows us to use 'populate' and reference the ObjectId to get the name, and price for the item, so in mongo it's stored as entries[{item_id: object id, status, comment}, {...}]. BUT, after populate we get {item_id: {name, price}, status, comment}, so I had to do some parsing (in toEntry) to get comment, status, name, and price to the same level. I'm not sure how else to model a many-to-many relationship, with additional fields in the join table (status, comment).